services/grafana-loki/0.33.2/minio.yaml

```diff
@@ -84,7 +84,7 @@ spec:
                 name: grafana-loki-minio
       containers:
       - name: minio-mc
-        image: minio/mc:RELEASE.2020-11-25T23-04-07Z
+        image: minio/mc:RELEASE.2022-04-07T21-43-27Z
```

Comment:
this one is quite a big jump just we should just make sure it's still running as expected

Reply:
+1

Reply:
Tested on kind, minio and loki are happy.